test/regress/cli/regress1/nl/issue9420-ll.smt2

```diff
@@ -0,0 +1,13 @@
+; COMMAND-LINE: --learned-rewrite
+; EXPECT: unsat
+; DISABLE-TESTER: unsat-core
+; DISABLE-TESTER: proof
```

Comment:
Are these necessary?

Reply:
Yes, we don't have proof support for `--learned-rewrite`, which impacts the correctness of unsat cores due to dependencies.